Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:3.4
+
+RUN apk -U add ca-certificates ruby ruby-bundler ruby-dev ruby-io-console ruby-irb ruby-rdoc ruby-json
+
+# Setup bundle user and directory
+RUN adduser -h /home/bundle -D bundle && \
+    mkdir -p /home/bundle && \
+    chown -R bundle /home/bundle
+
+# Copy the bundle source to the image
+WORKDIR /home/bundle
+COPY Gemfile Gemfile.lock /home/bundle/
+
+# Install Git, run Bundler, and uninstall Git to recover space
+RUN apk add git && \
+    su bundle -c 'bundle install --path .bundle' && \
```

Comment:
I think this'll also install `rspec`. You might need to add the `--without` flag: `bundle install --without development test`